editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/Wsdl2RestWizard.java

```diff
@@ -370,6 +408,59 @@ protected Path initContextForTool(boolean isBlueprint, boolean isSpringBoot, Fil
 		return contextpath;
 	}
 
+	/**
+	 * 
+	 * @param path
+	 * @param monitor
+	 */
+	protected void updateIDValueInCamelFile(Path path, IProgressMonitor monitor) {
```

Comment:
I like this method - a bit brute force, but simply opening the file and saving it is all we need.

Reply:
I don't like this method which is making editors blinking and changing the position of them in case several are open. It is also the root cause of not working when an xml editor is opened for the camel file.

Reply:
I tried to address your concerns. Please let me know if that fits better now.

Reply:
the model is not modified now in design view (i can see it is modified in the Project explorer) and when I switch to the Source tab, the previous model is overriding the generated one